docs/pages/machine-id/access-guides/terraform.mdx

```diff
@@ -173,6 +173,153 @@ Check your Teleport cluster, ensuring the role has been created:
 $ tctl get role/terraform-test
 ```
 
+## Example: Github Actions
+
+Let's look at this in a concrete example of running Terraform in CI using GitHub Actions.
+
+First we'll create a token and bot that allow jobs running in Actions in a specific organization,
+in a specific repository where we keep all our Terraform code, to use the Terraform role:
+
+```yaml
+---
+kind: token
+version: v2
+metadata:
+  name: github-actions-bot
+  description: Token for GitHub Actions bot for CI Terraform runs
+spec: 
+  roles:
+  - Bot
+  bot_name: github-actions
+  join_method: github
+  github:
+    allow:
+    - repository: <your org>/<your terraform repo>
+---
+kind: bot
+version: v2
+metadata:
+  name: github-actions
+spec:
+  token_id: github-actions-bot
+  roles:
+  - terraform
+```
+
+Now we'll combine the available `teleport-actions` and `hashicorp` Actions to run Terraform in CI, using
+tbot to authenticate with the Teleport cluster. The Actions `teleport-actions/setup` and `teleport-actions/auth`
+perform the steps we did in Step 2 above, configuring `tbot` and then starting it up in
+[one-shot mode](../../includes/machine-id/daemon-or-oneshot.mdx).
+
+```yaml
+on:
+ push:
+   branches:
+   - main
+   paths:
+   - terraform/**
+ pull_request:
+   branches:
+   - main
+   paths:
+   - terraform/**
+ 
+env:
+ # verbosity setting for Terraform logs
+ TF_LOG: INFO
+ # Credentials for deployment to AWS
+ AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+ AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+ # S3 bucket for the Terraform state
+ BUCKET_TF_STATE: ${{ secrets.BUCKET_TF_STATE}}
+
+jobs:
+  terraform:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Install Teleport
+        uses: teleport-actions/setup@v1
+        with:
+          version: 15.0.1
```

Comment:
Is there a variable we can use for this so that it stays up to date.

Reply:
Great idea. It looks like `(=teleport.version=)` would work but if someone who knows better than me could confirm I'd appreciate it.

Reply:
I've put that variable in for now.

Reply:
It works! Checked it in the preview.